Model/Sources.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Ampersand\DisableStockReservation\Model;
+
+use Magento\Framework\Model\AbstractModel;
+
+/**
+ * Class Sources
+ * @package Ampersand\DisableStockReservation\Model
+ */
+class Sources extends AbstractModel
```

Comment:
Can we use an interface for the sources model? The interface would have methods for `setSource`, `getSource`, `setOrderId` etc